activerecord/lib/active_record/relation.rb

```diff
@@ -101,6 +101,10 @@ def create(attributes = nil, &block)
       end
     end
 
+    def insert_all(attributes, returning: nil, unique_by: nil)
+      InsertAll.new(self, attributes, on_duplicate: :skip, returning: returning, unique_by: unique_by).execute
+    end
+
```

Comment:
After realizing you could pass `scope_for_association` to `InsertAll` from `Persistence` (👍), can we drop this method from `Relation`?